docs/UsersGuide/source/BuildRunSRW.rst

```diff
@@ -601,7 +633,55 @@ For WCOSS_CRAY:
    ``valid_param_vals script``. In addition, various example configuration files can be
    found in the ``regional_workflow/tests/baseline_configs`` directory.
 
+.. _VXConfig:
+
+Configure METplus Verification Suite (Optional)
+--------------------------------------------------
+
+Users who want to use the METplus verification suite to test their forecasts need to load the appropriate modules and add additional information to their ``config.sh`` file. Other users may skip to the :ref:`next section <SetUpPythonEnv>`. 
+
+.. note::
+   METplus is preinstalled on `Level 1 <https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers>`__ systems. METplus *installation* is currently not supported for this release of the SRW App, but METplus *use* is supported on systems with a functioning METplus installation. For more information about METplus, see :numref:`Section %s <MetplusComponent>`.
```

Comment:
Consider changing to: METplus *installation* is currently not **included as part of the build process** for this release of the SRW App